src/components/InLineNotification.tsx

```diff
@@ -69,8 +64,21 @@ export function InLineNotification({
 
       {(ctaLabel || ctaLabel2) && (
         <View style={[styles.row, styles.ctaRow]}>
-          {renderCtaLabel(ctaLabel, onPressCta, severityColor.primary)}
-          {renderCtaLabel(ctaLabel2, onPressCta2, severityColor.primary)}
+          {ctas.map(({ label, onPress }, index) => (
+            <Text
+              key={label}
+              style={[
+                styles.ctaLabel,
+                {
+                  color: severityColor.primary,
+                  marginRight: index === 0 ? Spacing.Small12 : 0,
```

Comment:
we can probably leverage the [gap](https://reactnative.dev/docs/layout-props#gap) property in container component instead

Reply:
TY!! i keep forgetting that we can use `gap` now (we weren't on the right version of react native for a long time). i tidied up the styles